public/partials/demand_new.html

```diff
@@ -63,8 +70,10 @@
         </md-input-container>
 
         <md-input-container layout="row" layout-sm="column" layout-align="center center">
-            <md-button ng-click="toggleDemandForm()">Cancel</md-button>
-            <md-button ng-click="saveDemand()">Save</md-button>
+            <md-button ng-if="new_demand" ng-click="toggleDemandForm()">Cancel</md-button>
```

Comment:
@nick22891 code reuse! remember don't place unnecessary object, function in your code.  Try your best to reuse objects.